bind_exporter.go

```diff
@@ -248,7 +310,7 @@ func histogram(stats []Stat) (map[float64]uint64, uint64, error) {
 
 func main() {
 	var (
-		listenAddress = flag.String("web.listen-address", ":9109", "Address to listen on for web interface and telemetry.")
+		listenAddress = flag.String("web.listen-address", ":9119", "Address to listen on for web interface and telemetry.")
```

Comment:
Quoting the commit message :)

> The port 9109 is already used by the graphite exporter. Use the next
free port listed in https://github.com/prometheus/prometheus/wiki/Default-port-allocations.